apps/web/components/eventtype/EventTypeSingleLayout.tsx

```diff
@@ -8,6 +8,7 @@ import type { UseFormReturn } from "react-hook-form";
 
 import useLockedFieldsManager from "@calcom/features/ee/managed-event-types/hooks/useLockedFieldsManager";
 import { useOrgBranding } from "@calcom/features/ee/organizations/context/provider";
+import { EventTypeEmbedButton, EventTypeEmbedDialog } from "@calcom/features/embed/EventTypeEmbed";
```

Comment:
Moved embed to features because it needs it's own type, different HOCs over Embed.tsx, utils and hooks.